lib/generators/suspenders/lint_generator.rb

```diff
@@ -0,0 +1,67 @@
+module Suspenders
+  module Generators
+    class LintGenerator < Rails::Generators::Base
+      include Suspenders::Generators::Helpers
+
+      source_root File.expand_path("../../templates/lint", __FILE__)
+      desc "Creates a holistic linting solution that covers JavaScript, CSS, Ruby and ERB."
+
+      def install_dependencies
+        run "yarn add stylelint eslint @thoughtbot/stylelint-config @thoughtbot/eslint-config --dev"
```

Comment:
I got lost in the Yarn vs NPM battle. Do Rails projects default to Yarn now?

Reply:
Rails actually [defaults to Import Maps](https://guides.rubyonrails.org/working_with_javascript_in_rails.html#choosing-between-import-maps-and-a-javascript-bundler), but with the introduction of #1145 we've opted into use a JavaScript Runtime.

 However, it seems Rails still favors `yarn` over `npm` based on how they install packages in cssbundling-rails:

https://github.com/rails/cssbundling-rails/blob/66635235a96909f688627789b40cc81795917f7f/lib/install/helpers.rb#L4-L6